client/pages/render-exercises.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import RenderSet from './render-set';
+
+export default class RenderExercises extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      setCounter: [0]
+    };
+
+    this.appendSetCounter = this.appendSetCounter.bind(this);
+  }
+
+  appendSetCounter() {
+    const newArray = this.state.setCounter;
+    newArray.push(0);
```

Comment:
Careful, this is just a reference and not a copy. You can use Array.prototype.slice() to create a copy.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice